### PR TITLE
Add DeactivateApp wrapper method and test case

### PR DESCRIPTION
**Pull request by kamundson-va**
[GLO-1744](https://vendasta.jira.com/browse/GLO-1744)

This adds the DeactivateApp wrapper method to the AccountServiceClient. I still have to work on getting the tests to run to validate this. 

@vendasta/lucky-charms

### CODE REVIEW

**Review by emayukos — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/composer.lock
+/vendor/
+/.idea/
```
Comment (ParkerReese) [inline .gitignore]: anyone know why things like new lines at the end of files?

File: tests/AccountServiceClientTest.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new AccountsServiceClient($environment);
+
+        $req = new DeactivateAppRequest();
+        $req->setBusinessId("AG-P3KH8VTHR5");
+        $req->setAppId("MP-MCLBF8TMSMLCQ5W72LNHQV5XVW4KC2MX");
```
Comment (ParkerReese) [inline tests/AccountServiceClientTest.php]: No one else running this test will have it pass with these values. Could maybe leave them open ended like `MP-EXAMPLE`, just to signify that

File: tests/AccountServiceClientTest.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        $req->setActivationId("ea6f6da0-ad68-49d6-8439-8a693fdaea20");
+        $req->setDeactivationType(DeactivationType::DEACTIVATION_TYPE_CANCEL);
+
+        $resp = $client->DeactivateApp($req);
+    }
```
Comment (ParkerReese) [inline tests/AccountServiceClientTest.php:35]: Doesn't look like you're asserting anything in this test. I thought php unit got mad at tests without asserts

**Review by ParkerReese — APPROVED** [review]
Just some thoughts. LGTM

**Pull request merged**